packaging/request-a-package-update/en.md

```diff
@@ -0,0 +1,20 @@
++++
+title = "Requesting a Package update"
+lastmod = "2018-06-25T23:13:29+02:00"
++++
+# Requesting A Package Update 
+
+Packages updates are typically provided by community or dedicated package maintainers, however if we are not shipping the latest stable release of a package, you can let us know using our Task Tracker.
+
+**Please look to see if an update request has been filed for the software or library you require**. If there isn't an existing request, you can use the template below for requesting to update a package.
+
+Please provide the following information:
+
+- Title of Task: Update $packagename to $version
+- Description: Explanation as to the value-add of updating this package. 
+- Updates solely for a new version number, platform-specific updates (such as to macOS or Windows) which aren't relevant, or solely translation updates, are generally not accepted or not prioritized.
+- Link to source tarball/zip file
+- master.zip files **are not permitted**. We require versioned tarballs, for example: "1.2.3.tar.gz".
+- Use the tag **Software** for tagging your update request
+
+Please put this information into a Bug Report (create via "Create Task") on our Developer Portal via the Maniphest section on https://dev.solus-project.com/
```

Comment:
Should ideally be: `Please put this information into a new [task](https://dev.solus-project.com/maniphest/task/edit/form/1/).`